js/north.js

```diff
@@ -44,6 +44,33 @@
 			} );
 		};
 
+		$.fn.smoothScroll = function () {
+			$( this ).click( function ( e ) {
+				var $a = $( this );
+				var $target = $( '[name=' + this.hash.slice( 1 ) + ']' ).length ? $( '[name=' + this.hash.slice( 1 ) + ']' ) : $( $a.get( 0 ).hash );
+
+				if ( $target.length ) {
+
+					var height = 0;
+					if ( $( '#masthead' ).hasClass( 'sticky-menu' ) ) {
+						height += $( '#masthead' ).outerHeight();
+					}
+					if ( $( 'body' ).hasClass( 'admin-bar' ) ) {
+						height += $( '#wpadminbar' ).outerHeight();
+					}
+
```

Comment:
As a general style comment, this should just be a single blank line - 3 line breaks is a bit too many.